### PR TITLE
Abraham development

### PR DESCRIPTION
**Pull request by abrahamharos**

### CODE REVIEW

**Review by mschillaci — APPROVED** [review]
Looks good to me!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
+This repo contains abraham's portfolio and projects of the first half of STEP.
```
Comment (akihiroota87) [inline README.md]:
```suggestion
This repo contains Abraham's portfolio and projects for ...
```
(Capitalize your first name and change "of" to "for")

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,9 +8,33 @@
...
+			</header>
+			<article>
+      <h2>About me.</h2>
+				<p>Since childhood I have been passionate about software development, 
+					in middle school I was  interested on learning how a computers works, 
+					so I decided to enroll in an introduction to web development course, 
+					that was a very satisfying experience and made me decide for more knowledge 
+					in the field, hence in high school I enter to a software development program 
+					where I learned many things, such as Object Oriented Programming, mobile app 
+					development and database management.
```
Comment (akihiroota87) [inline portfolio/src/main/webapp/index.html]:
```suggestion
					development and database management.</p>
```
Most html elements require an opening and closing tag. The closing paragraph tag is optional, but it's a good habit to use closing tags to make sure your document is valid.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -8,9 +8,33 @@
...
+				<h1>Abraham Haros's Portfolio</h1>
+			</header>
+			<article>
+      <h2>About me.</h2>
+				<p>Since childhood I have been passionate about software development, 
+					in middle school I was  interested on learning how a computers works, 
+					so I decided to enroll in an introduction to web development course, 
+					that was a very satisfying experience and made me decide for more knowledge 
+					in the field, hence in high school I enter to a software development program 
+					where I learned many things, such as Object Oriented Programming, mobile app
```
Comment (akihiroota87) [inline portfolio/src/main/webapp/index.html]:
```suggestion
					where I learned many things, such as object-oriented programming, mobile app 
```
No need to capitalize this.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -8,9 +8,33 @@
...
+					in middle school I was  interested on learning how a computers works, 
+					so I decided to enroll in an introduction to web development course, 
+					that was a very satisfying experience and made me decide for more knowledge 
+					in the field, hence in high school I enter to a software development program 
+					where I learned many things, such as Object Oriented Programming, mobile app 
+					development and database management.
+				<p>For me, applying the knowledge to real life problems is the best way to 
+					learn about something, hence why I started to make my first personal projects 
+					on high school. I have worked in solutions for my classes, for example, 
+					implementing c++ functions to evaluate physics functions.
```
Comment (akihiroota87) [inline portfolio/src/main/webapp/index.html]:
```suggestion
					implementing C++ functions to evaluate physics functions.</p>
```
Same thing here :) (Also capitalize "C++")

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -8,9 +8,33 @@
...
+				<p>The first project that I worked in, was a webpage for a local business, I worked on 
+					the backend of the contact form and newsletter. Then I worked in a project of administration 
+					of a mechanical workshop, there I developed it with databases and PHP. Also I worked on 
+					a logistic platform for shipments in an industrial warehouse, that was one of my most 
+					challenging project because I have to learn the logistic process of shipments and implement 
+					that in the platform as well as create the database relations and finally I learn about 
+					servers in order to implement the software in cloud.
```
Comment (akihiroota87) [inline portfolio/src/main/webapp/index.html]:
```suggestion
					servers in order to implement the software in cloud.</p>
```

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -8,9 +8,33 @@
...
+				<p>My strongest programming language is C++, since all my university courses have been taught 
+					in that language, I am learning data structures and I find myself more comfortable using C++ 
+					than other languages, also I have participated in competitive programming contest and I use C++ too.
```
Comment (akihiroota87) [inline portfolio/src/main/webapp/index.html]:
```suggestion
					than other languages, also I have participated in competitive programming contest and I use C++ too.</p>
```

**Review by akihiroota87 — APPROVED** [review]
Looks good to me! Please address the comments I added, but then you should be good to merge. Thanks!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (abrahamharos) [reply portfolio/src/main/webapp/index.html]: Of course, thank you for the tip.

**Pull request merged**